libraries/wallet/operation_printer.cpp

```diff
@@ -146,6 +150,19 @@ std::string operation_printer::operator()(const htlc_redeem_operation& op) const
    return fee(op.fee);
 }
 
+std::string operation_printer::operator()(const htlc_redeemed_operation& op) const
+{
+   out << "Redeem HTLC with database id "
+         << std::to_string(op.htlc_id.space_id)
+         << "." << std::to_string(op.htlc_id.type_id)
+         << "." << std::to_string((uint64_t)op.htlc_id.instance)
+         << " with preimage \"";
+   for (unsigned char c : op.preimage)
+      out << c;
```

Comment:
* I think a hex dump is better.
* I think it's not good to print the whole image directly in the history commands since likely it's too long thus not pretty.

 BTW the same issues apply to the printer for `htlc_redeem_operation`.

Reply:
The whole image is necessary for those looking to redeem on another chain. Agreed, a hex dump would make more sense. Will fix.

Reply:
The whole image is always retrieve-able via API calls. For displaying directly in CLI, I think it's acceptable if it's only a few lines. But it can be as large as hundreds of kilobytes. For these large images, IMHO we can just show a message to indicate it's too large so not fully displayed.